### PR TITLE
Renominate proposals UI

### PR DESCRIPTION
**Pull request by dwt**
Add ui to renominate proposals.

ping @joka, there's also a backend fix in this, please review.

### CODE REVIEW

**Comment by joka** [comment]
>  @joka, there's also a backend fix in this, please review.

lgtm

@dwt fixed in https://github.com/liqd/adhocracy3/pull/2643/commits/f7aef34a1a2bfdac6540b963d55245d4bc70f16c the backend code to renominated proposals. The function 'index_rates' was called with missing default argument. The tests should have fail in this case, but did not due tu missing ( autospec=True parameter ). The tests are fixed in #https://github.com/liqd/adhocracy3/pull/2643/commits/e3ee5e6aa3484b94d2fcf44ecd9068b57cf4eafe ( ping @2e2a )

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Process.ts [hunk 1]
```diff
@@ -112,13 +112,15 @@ export var workflowSwitchDirective = (
             });
 
             scope.switchState = (newState) => {
+                if ( ! $window.confirm("Will switch to process state " + newState + ". (Page will reload)")) {
+                    return;
+                }
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Process.ts]: good idea!

**Comment by der-john** [comment]
small question: How do I select / reject proposals in the UI?

**Comment by dwt** [comment]
To select/reject: vote for proposals, then advance the process (login as admin to see the button besides the process state indicator).

**Comment by 2e2a** [comment]
backend looks good

**Comment by der-john** [comment]
frontend, too, but some strings need translation. I'll take care of it.

**Comment by dwt** [comment]
> frontend, too, but some strings need translation. I'll take care of it.
> 
Thanks! :)

**Pull request merged**